### PR TITLE
{tools}[intel/2015b] deap 0.9.2 w/ Python 2.7.10 (REVIEW)

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.10
See https://gist.github.com/ddcb3f35f382e1850be0 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux centos linux 7.1.1503, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/073ba8fa2c73eb7e6866 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.10
See https://gist.github.com/a325a1c1ebc367965f93 for a full test report.

File: easybuild/easyconfigs/d/deap/deap-0.9.2-intel-2015b-Python-2.7.10.eb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+name = 'deap'
+version = '0.9.2'
+
+homepage = 'http://deap.readthedocs.org/en/master/'
+description = """DEAP is a novel evolutionary computation framework for rapid prototyping and testing of ideas.
+ It seeks to make algorithms explicit and data structures transparent."""
+
+toolchain = {'name': 'intel', 'version': '2015b'}
+
+source_urls = [PYPI_SOURCE]
```
Comment (ehiggs) [inline easybuild/easyconfigs/d/deap/deap-0.9.2-intel-2015b-Python-2.7.10.eb]: :+1: Is this new? I've wanted something like this for a while.

File: easybuild/easyconfigs/d/deap/deap-0.9.2-intel-2015b-Python-2.7.10.eb [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+sources = [SOURCE_TAR_GZ]
+
+pyver = '2.7.10'
+versionsuffix = '-Python-%s' % pyver
+dependencies = [('Python', pyver)]
+
+pyshortver = '.'.join(pyver.split('.')[:2])
```
Comment (ehiggs) [inline easybuild/easyconfigs/d/deap/deap-0.9.2-intel-2015b-Python-2.7.10.eb]: Actual code in a config. :(
You could simplify this with `pyver[:3]` or `py_maj_min = '2.7'; pyver = '%s.10' % py_maj_min` and then use `py_maj_min` here.

File: easybuild/easyconfigs/d/deap/deap-0.9.2-intel-2015b-Python-2.7.10.eb (hunk 1 above)
Comment (boegel) [inline easybuild/easyconfigs/d/deap/deap-0.9.2-intel-2015b-Python-2.7.10.eb]: It's been there for quite a while already (Dec'13, see https://github.com/hpcugent/easybuild-framework/pull/793)

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.10
See https://gist.github.com/c38e4e80a9810a77160b for a full test report.

**Comment by boegel** [comment]
Thanks for the review @ehiggs!

**Pull request merged**